benchmark/misc/arguments.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const { createBenchmark } = require('../common.js');
+const { Writable } = require('stream');
+const { format, inherits } = require('util');
+
+const methods = [
+  'restAndSpread',
+  'argumentsAndApply',
+  'restAndApply',
+  'predefined'
+];
+
+const bench = createBenchmark(main, {
+  method: methods,
+  n: [1e6]
+});
+
+const nullStream = createNullStream();
```

Comment:
Unrelated but can't this be simplified to just `new Writable({ write: () => {} })`?

Reply:
I went ahead and removed the fake stream overall as it had no real purpose in this benchmark.